airflow/contrib/operators/gcs_download_operator.py

```diff
@@ -33,6 +33,9 @@
 class GoogleCloudStorageDownloadOperator(BaseOperator):
     """
     Downloads a file from Google Cloud Storage.
+    If a filename is supplied, it writes the file to the specified location, alternatively one can
```

Comment:
```suggestion

    If a filename is supplied, it writes the file to the specified location, alternatively one can
```

Need a blank line between the "summary" and body of the doc.